### ISSUE TITLE
drag and drop

### ISSUE THREAD

**Issue by olimart** [olimart/kanban#1]
Hey,

drag and drop is a big thing in my app.

- check out dragula https://github.com/bevacqua/dragula or http://rubaxa.github.io/Sortable/
- as light as possible (no jQuery UI for example)
- in data add `position` to sample data
  - sort card and list by position
- ability to drag around lists
  - new list form should always remain last
- ability to drag cards
  - within list
  - in other list
  - new card form must remain last
- must trigger event so we can sync position to server
- basically just like troll does

**Comment by anishdcruz** [comment]
I think basic html 5 drag and drop feature will be sufficient. So it will be light weight.
http://www.html5rocks.com/en/tutorials/dnd/basics/

**Comment by olimart** [comment]
makes sense actually.

**Comment by anishdcruz** [comment]
There is a Vuejs plugin for drag and drop. But it is limited in functionality.

https://www.npmjs.com/package/vue-dnd

**Comment by olimart** [comment]
interesting.

**Comment by olimart** [comment]
I see you've pushed. Let me know when I can review d&d of cards.

**Comment by olimart** [comment]
@anishdcruz Take a look, sortable is also easy to use https://jsfiddle.net/2j1hwy9m/

**Comment by anishdcruz** [comment]
That looks good, 
I am stuck, in moving card to another list. That's the difficult part.

**Comment by olimart** [comment]
feel free to use dragula or sortable (multi) if easier.
Both are light and don't require jQuery

I'll push some stuff shortly
